v2/internal/controllers/insights_diagnosticsetting_crud_v20210501preview_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	insights "github.com/Azure/azure-service-operator/v2/api/insights/v1api20210501preview"
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1api20201101"
+	storage "github.com/Azure/azure-service-operator/v2/api/storage/v1api20210401"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_Insights_DiagnosticSetting_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+	rg_name := rg.Status.Name
+
+	// vnet := VMVirtualNetwork(tc, testcommon.AsOwner(rg))
+
+	vnet := &network.VirtualNetwork{
+		ObjectMeta: tc.MakeObjectMetaWithName(tc.Namer.GenerateName("vn")),
```

Comment:
There's a helper method that does exactly this.

```suggestion
		ObjectMeta: tc.MakeObjectMeta("vn"),
```

Reply:
Updated with the suggested changes.